TADashboard/src/main/java/page_objects/DataProfilesDisplayFieldsPage.java

```diff
@@ -0,0 +1,23 @@
+package page_objects;
+
+import com.logigear.control.common.imp.CheckBox;
+import com.logigear.control.common.imp.Label;
+import utils.common.Constants;
+
+import java.util.List;
+
+public class DataProfilesDisplayFieldsPage extends BasePage {
+    private Label lblProfileHeader = new Label("css=#profilesettings > tbody > tr > td.profilesettingheader");
+    private Label lblDisplayFieldsLabels = new Label("css=#profilesettings > tbody label");
```

Comment:
you already have lbl, why do you need labels at the end?

Reply:
Yeah, I have removed it.